package.json

```diff
@@ -108,7 +108,7 @@
     "browserify": "16.2.3",
     "cross-env": "5.2.0",
     "del": "3.0.0",
-    "electron": "6.1.5",
+    "electron": "6.1.7",
```

Comment:
Why do we need to update the electron version for this?

Reply:
You are right, it's not relevant for this PR. But, since `6.1.7` has some important fixes, I thought of bundling it with this in the original PR.

@johankwarnmarksymphony Should I raise a separate PR for the upgrade or is it fine here?

Reply:
I've reverted back to `6.1.5` as it makes sense to not bundle an electron upgrade with this PR.

Thanks for pointing out @mattias-symphony :)